client/src/main/java/org/schlunzis/kurtama/client/service/impl/SessionService.java

```diff
@@ -33,6 +30,7 @@ public class SessionService implements ISessionService {
 
     private final ObservableList<LobbyInfo> lobbyList = FXCollections.observableList(new ArrayList<>());
     private final ObservableList<String> chatMessages = FXCollections.observableList(new ArrayList<>());
+    private final ObservableList<String> lobbyUsersList = FXCollections.observableList(new ArrayList<>());
```

Comment:
Wir sollten den SessionService in aufteilen in ChatService, LobbyService... In einem anderen Issue.

Reply:
Finde ich gut